jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/steps/LambdaWorkflows.kt

```diff
@@ -128,7 +128,7 @@ fun createDeployWorkflow(
     Files.createDirectories(buildDir)
 
     return StepWorkflow(
-        BuildLambda(templatePath, buildDir, envVars, SamOptions(buildInContainer = useContainer)),
+        BuildLambda(templatePath, null, buildDir, envVars, SamOptions(buildInContainer = useContainer)),
```

Comment:
Nit: this seems to be the only `BuildLambda` withoutnamed arguments

Reply:
Made all of these workflow builds consistent looking with named args and sep lines.